slasher/rpc/detect_update_min_max_span_test.go

```diff
@@ -1,299 +0,0 @@
-package rpc
```

Comment:
tests moved to slasher/detection/attestations/detect_surround_surrounded_test.go